var/spack/repos/builtin/packages/davix/package.py

```diff
@@ -14,8 +14,10 @@ class Davix(CMakePackage):
     list_url = "http://grid-deployment.web.cern.ch/grid-deployment/dms/lcgutil/tar/davix/"
     list_depth = 1
 
+    version('0.7.3', sha256='cd46276e72c6a0da1e2ad30eb66ec509a4c023687767c62a66713fa8c23d328a',
+            url='https://grid-deployment.web.cern.ch/grid-deployment/dms/lcgutil/tar/davix/0.7.3/davix-0.7.3.tar.gz')
     version('0.6.8', sha256='e1820f4cc3fc44858ae97197a3922cce2a1130ff553b080ba19e06eb8383ddf7',
-            url='http://grid-deployment.web.cern.ch/grid-deployment/dms/lcgutil/tar/davix/0.6.8/davix-0.6.8.tar.gz')
+            url='https://grid-deployment.web.cern.ch/grid-deployment/dms/lcgutil/tar/davix/0.6.8/davix-0.6.8.tar.gz')
```

Comment:
Any idea why this package uses version-specific URLs? I don't think they're necessary.

Reply:
Maybe @gartung remembers?

Reply:
I don't recall. @JavierCVilla might know. Maybe didn't now about url_for_version(sp?) function.

Reply:
done.

Reply:
@iarspider Note that `url_for_version` is only used if the `url` attribute is omitted from the `version()` call.